ieeg_visual_multimodal/sub-som682/ses-nyu3t01/fmap/sub-som682_ses-nyu3t01_dir-RL_epi.json

```diff
@@ -0,0 +1,165 @@
+{
+    "AcquisitionDateTime": "2018-07-10T10:31:40.190000", 
+    "AcquisitionMatrixPE": 96, 
+    "AcquisitionNumber": 1, 
+    "AcquisitionTime": "10:31:40.190000", 
+    "BandwidthPerPixelPhaseEncode": 16.801, 
+    "BaseResolution": 96, 
+    "BodyPartExamined": "BRAIN", 
+    "ConsistencyInfo": "N4_VE11C_LATEST_20160120", 
+    "ConversionSoftware": "dcm2niix", 
+    "ConversionSoftwareVersion": "v1.0.20180622 (JP2:OpenJPEG) GCC4.8.4", 
+    "DerivedVendorReportedEchoSpacing": 0.000620003, 
+    "DeviceSerialNumber": "166018", 
+    "DwellTime": 2.5e-06, 
+    "EchoTime": 0.0648, 
+    "EchoTrainLength": 96, 
+    "EffectiveEchoSpacing": 0.000620003, 
+    "FlipAngle": 90, 
+    "ImageOrientationPatientDICOM": [
+        1, 
+        0, 
+        -0.000174533, 
+        -0.000174533, 
+        0, 
+        -1
+    ], 
+    "ImageType": [
+        "ORIGINAL", 
+        "PRIMARY", 
+        "M", 
+        "ND", 
+        "MOSAIC"
+    ], 
+    "InPlanePhaseEncodingDirectionDICOM": "ROW", 
+    "InstitutionAddress": "Washington_Place_6_NYC_NYC_US_10003", 
+    "InstitutionName": "NYU_-_CBI", 
+    "InstitutionalDepartmentName": "Department", 
+    "IntendedFor": [
+        "ses-nyu3t01/func/sub-som682_ses-nyu3t01_task-spatialobject_run-1_events.tsv", 
```

Comment:
are the _events.tsv files supposed to be here?

Reply:
See above